src/modules/live-execution-tracker/services/live-execution-tracker.service.ts

```diff
@@ -179,6 +179,19 @@ export class LiveExecutionTrackerService implements ILiveExecutionTrackerService
     return elementsWithTokenHistory;
   }
 
+  private async getElementsWhereProcessWasStopped(processInstanceId: string): Promise<Array<IShape>> {
+    const flowNodeInstances: Array<DataModels.FlowNodes.FlowNodeInstance> =
+      await this._liveExecutionTrackerRepository.getFlowNodeInstancesForProcessInstance(processInstanceId);
+
+    return flowNodeInstances
+      .filter((flowNodeInstance: DataModels.FlowNodes.FlowNodeInstance) => {
+        return flowNodeInstance.state === 'error';
+      })
+      .map((flowNodeInstance: DataModels.FlowNodes.FlowNodeInstance) => {
+      return this._elementRegistry.get(flowNodeInstance.flowNodeId);
```

Comment:
Indentation is off.
Please fix to make this easier to read and understand.

```suggestion
        return this._elementRegistry.get(flowNodeInstance.flowNodeId);
```